ui-kit/src/main/kotlin/org/p2p/uikit/natives/UiKitSnackbar.kt

```diff
@@ -0,0 +1,74 @@
+package org.p2p.uikit.natives
+
+import android.view.View
+import com.google.android.material.snackbar.Snackbar
+
+fun interface SnackbarActionButtonClickListener {
+    fun onActionButtonClicked(clickedSnackbar: Snackbar)
+}
+
+fun View.showSnackbarShort(snackbarText: CharSequence) {
+    internalMakeSnackbar(
+        this,
+        text = snackbarText,
+        buttonText = null,
+        buttonAction = null,
+        duration = Snackbar.LENGTH_SHORT
+    )
+        .show()
+}
+
+fun View.showSnackbarShort(
+    snackbarText: CharSequence,
+    snackbarActionButtonText: CharSequence,
+    snackbarActionButtonListener: SnackbarActionButtonClickListener
+) {
+    internalMakeSnackbar(
+        this,
+        text = snackbarText,
+        buttonText = snackbarActionButtonText,
+        buttonAction = snackbarActionButtonListener,
+        duration = Snackbar.LENGTH_SHORT
+    )
+        .show()
+}
+
+fun View.showSnackbarLong(snackbarText: CharSequence) {
```

Comment:
I think duration can be optionally, and there is no reasons to duplicate methods just for this ))